### DISCUSSION TITLE
我的原神上班提醒小助手被 disable 了

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#847] [General]
![image](https://user-images.githubusercontent.com/48074861/177759681-7ad68ec5-be85-4949-95ea-28b1c6093544.png)

？？？

**Comment by Arondight** [comment]
发个邮件问问吧

**Comment by Xm798** [comment]
Hi Xm798,

Thanks for reaching out. The repository was actioned following reports of conduct in violation of the following prohibition found in our [Acceptable Use Policies](https://docs.github.com/site-policy/acceptable-use-policies/github-acceptable-use-policies):

    We do not allow content or activity on GitHub that is:

        using GitHub as a platform for propagating abuse on other platforms;

Please also note our additional terms related to Actions usage:

https://docs.github.com/site-policy/github-terms/github-terms-for-additional-products-and-features#actions

Specifically, the content or activity that was reported included

any activity that places a burden on our servers, where that burden is disproportionate to the benefits provided to users
if using GitHub-hosted runners, any other activity unrelated to the production, testing, deployment, or publication of the software project associated with the repository where GitHub Actions are used.

which we found to be in violation of our Acceptable Use Policies.

Because of the nature and volume of the prohibited activity, we will not be reinstating your repository.

Regards,
GitHub Trust & Safety

**Comment by Xm798** [comment]
我的 Action 只用来构建 docker 和同步 Gitee，这竟然是违法 TOS 的么

**Reply by Arondight** [reply]
曾经这个项目有过同步问题，我当时本着 action 只进不出的原则，没加，现在的 action 不对外输出任何东西

**Reply by Xm798** [reply]
但同步 Gitee 用的也是 Action market 里的 action，我看了有不少上千 star 的 repo 都在用。
至于输出，按他的说法我构建 docker 发布到 dockerhub 也是输出，那如果这算违规的话，违规的 repo 可就多了。
我把同步 Gitee 去掉了，留了构建 docker。

**Reply by Arondight** [reply]
是不是你的 fork 很多在建 docker ，触发了什么限制了

**Reply by Xm798** [reply]
> 是不是你的 fork 很多在建 docker ，触发了什么限制了

那应该不会，只有 push tag 才会触发构建，而且没有配置 token 的话即使触发了构建也会报错，不会进行下去。
所以就，很奇怪。

而且看邮件，还是被举报了，就更离谱了。

**Reply by Arondight** [reply]
> 而且看邮件，还是被举报了

不会官方又下场了吧，你这项目唯一有损害的就是米游社团队，官方的离谱操作可不止一两次了

**Reply by Xm798** [reply]
> 不会官方又下场了吧，你这项目唯一有损害的就是米游社团队，官方的离谱操作可不止一两次了

不懂啊……官方下场的话，一堆上千 star 能查体力的机器人优先级不应该比我高？……难道吃了名字里带 Genshin 的亏？

进一步问哪个 action 违反了 tos，结果只有机器人回复

> Hello,
> 
> Kindly see below:
> 
> Specifically, the content or activity that was reported included
> 
> any activity that places a burden on our servers,
> if using GitHub-hosted runners, any other activity unrelated to the production, testing, deployment, or publication of the software project associated with the repository where GitHub Actions are used.
> 
> which we found to be in violation of our Acceptable Use Policies.
> 
> Because of the nature and volume of the prohibited activity, we will not be reinstating your repository.
> 
> Regards,
> GitHub Trust & Safety

**Comment by mark9804** [comment]
这个是类似签到项目吗？那确实是会被gank的

**Reply by Xm798** [reply]
不是，就是检查实时便笺推送的

**Reply by mark9804** [reply]
还有一种可能性，就是你的名字里面有 helper 

别问，问就是玄学

**Comment by Arondight** [comment]
我知道你为什么会被封了，github 认为你在借助他的平台宣传腾讯云函数，之前有个很流行的签到项目，第一次因为 action 没了，第二次因为宣传云函数没了，后来迁移到 gitlab 了

**Reply by Xm798** [reply]
啊这？？这也行……未曾设想的道路

**Reply by Xm798** [reply]
怎么被巨硬收购之后这么离谱……

**Reply by Arondight** [reply]
> 怎么被巨硬收购之后这么离谱……

所以这个项目里只有项目本身的文档，其他有的没的都在 issue 里

**Reply by Arondight** [reply]
https://github.com/Xm798/Genshin-Dailynote-Helper/blob/2433febaa46b1bb2f5e1f022fde8a49d4a9e23fc/docs/README_ZH.md#腾讯云云函数-scf

我特意看了一下，你这个图文并茂的教程，官方很可能直接做软广告处理了，建议不要手把手的教，或者贴博客链接，别直接图文一大片放在项目里

**Reply by Arondight** [reply]
如果这个项目里写了篇图文教程，让大家使用项目前先去某某平台申请 vps ，怎么创建虚拟机，怎么设置啥的，估计被举报了也会没，以前没人管这些，后来 github 就变得比较专横了

**Reply by Xm798** [reply]
> 建议不要手把手的教，或者贴博客链接，别直接图文一大片放在项目里

可以，谢谢老大哥指点哈哈哈哈，我找个时间把文档迁出去，太离谱了
